Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -108,11 +108,7 @@ - (void)start
         dispatch_queue_t queue = dispatch_queue_create("me.dvor.objcTox.OCTToxAVQueue", NULL);
         self.timer = dispatch_source_create(DISPATCH_SOURCE_TYPE_TIMER, 0, 0, queue);
 
-        /**
-         * Call quality is not great without dividing the interval by 1/20th. This can
-         * be removed once toxav fixes this issue.
-         */
-        uint64_t interval = _toxav_iteration_interval(self.toxAV) * (NSEC_PER_SEC / 1000) / 20;
+        uint64_t interval = _toxav_iteration_interval(self.toxAV) * (NSEC_PER_SEC / 1000);
```

Comment:
Yep with this it is. There was nothing wrong with toxav before. I just didn't properly check for `toxav_iterate_interval`